### PR TITLE
docs(enums): Correct grammatical errors.

### PR DESCRIPTION
**Pull request by i0bs**
## What type of pull request is this?

- [X] Non-breaking code change
- [ ] Breaking code change
- [X] Documentation change/addition

## Description
I looked at some of the documentation for your enumerable objects and found that you had grammatical inconsistencies in the word usage. This PR aims to correct that.

## Changes
* Used proper English grammar.

## Checklist

- [X] I've formatted my code with [Black](https://black.readthedocs.io/en/stable/)
- [X] I've ensured my code works on `Python 3.10.x`
- [X] I've tested my code

(I didn't test it at all but let's go with that 😉)

### CODE REVIEW

File: dis_snek/models/enums.py [hunk 1]
```diff
@@ -534,7 +534,7 @@ class ActivityType(IntEnum):
 
 
 class Status(str, Enum):
-    """Represents the statuses a user may have."""
+    """Represents the status' a user may have."""
```
Comment (silasary) [inline dis_snek/models/enums.py]: Errant apostrophe

**Review by silasary — CHANGES_REQUESTED** [review]

File: dis_snek/models/enums.py (hunk 1 above)
Comment (i0bs) [reply dis_snek/models/enums.py]: Disagree, it's valid in punctuation and in it's contextually derived use it's proper for plural representation of how many status is possessed.

Comment (LordOfPolls) [reply dis_snek/models/enums.py]: In English statuses is acceptable as per the Oxford Dictionary

Comment (i0bs) [reply dis_snek/models/enums.py]: Well I think Oxford is wrong >:(

Comment (LordOfPolls) [reply dis_snek/models/enums.py]: This PR wont be merged until this is reverted

Comment (i0bs) [reply dis_snek/models/enums.py]: ok ok fine :(

**Review by silasary — APPROVED** [review]

**Pull request merged**